attorch/train.py

```diff
@@ -31,17 +31,25 @@ def early_stopping(model, objective, interval=5, patience=20, start=0, max_iter=
         max_iter:  maximum number of iterations before the iterator terminated
         maximize:  whether the objective is maximized of minimized
     """
+    training_status = model.training
+
+    def _objective(mod):
```

Comment:
What if, for some reason, some part of the model was set to `train=True` and the other to `train=False`